toolkit/ar/src/main/java/com/arcgismaps/toolkit/ar/internal/GeospatialTrackingCameraController.kt

```diff
@@ -170,29 +164,64 @@ internal class GeospatialTrackingCameraController(
      *
      * @since 200.7.0
      */
-    private fun checkForEarthStateErrors(earth: Earth) {
+    private fun checkForEarthStateErrors(earth: Earth, hasSetOriginCamera: Boolean) {
+        if (earth.earthState != EarthState.ENABLED && error != null) {
+            // if we are in an error state and the earth state is not enabled,
+            // then don't do anything.
+            // This prevents us from changing an error that might already exist and propagating that
+            // to the user, but it's probably better than propagating a new error every frame even
+            // if it hasn't changed.
```

Comment:
Slight rewording to avoid confusion:
```suggestion
            // to the user, which is better than propagating a new error every frame even
            // if it hasn't changed.
```